### DISCUSSION TITLE
How to has brotli compression by default on vercel?

### DISCUSSION THREAD

**Discussion by pedromcunha** [vercel/next.js#36688] [Help]
Hi, I have a nextjs app that’s being served by vercel. How do I go about changing the default compression for the js chunks (I believe it’s gzip) to brotli?

**Comment by ijjk** [comment]
Hi, the compression on Vercel will be based on the `Accept-Encoding` header sent in the request, if the header specifies brotli (br) is supported it will be used over gzip.

**Reply by pedromcunha** [reply]
Right, I understand that but how to change the accept encoding headers sent for the javascript chunks? Right now it sends "gzip, deflate, br"

**Reply by SukkaW** [reply]
Vercel will always prefer brotil even if you have sent `gzip, deflate, br`. Just check the response header `Content-Encoding`.

**Comment by tapz** [comment]
How do I use next.js brotli implementation in apis?
